.github/workflows/test.yml

```diff
@@ -15,46 +12,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        package:
-          [
-            core-contracts,
-            core-utils,
-            core-modules,
-            sample-project,
-            main,
-            oracle-manager,
-            spot-market,
-            perps-market,
-            core-subgraph,
-            spot-market-subgraph,
-            perps-market-subgraph,
+        workspace: [
+            "@synthetixio/main",
+            "@synthetixio/oracle-manager",
+            # "@synthetixio/governance", # no tests :/
+
+            "@synthetixio/core-contracts",
+            "@synthetixio/core-utils",
+            "@synthetixio/core-modules",
+            "@synthetixio/hardhat-storage",
+            "@synthetixio/sample-project",
+
+            # "@synthetixio/legacy-market", # tests fail
```

Comment:
Tests shouldn't fail, what can I say. The tests were refreshed just a couple weeks ago.